academy/4-my-own-chain/can-play.md

```diff
@@ -144,15 +144,199 @@ Now you need to implement the answer to the player's query in `grpc_query_can_pl
     }
     ```
 
-6. If all went fine:
+6. If all went well:
 
-    ```go [https://github.com/cosmos/b9-checkers-academy-draft/blob/b53297d8e87e31b1fc7fb839fce527e66a2a0116/x/checkers/keeper/grpc_query_can_play_move.go#L79-L82]
+    ```go [https://github.com/cosmos/b9-checkers-academy-draft/blob/0f56961/x/checkers/keeper/grpc_query_can_play_move.go#L79-L82]
     return &types.QueryCanPlayMoveResponse{
         Possible: true,
         Reason:   "ok",
     }, nil
     ```
 
+## Interact via the CLI
+
+A friendly reminder that the CLI can always inform you about the available commands:
+
+<CodeGroup>
+<CodeGroupItem title="Checkers" active>
+
+```sh
+$ checkersd query checkers --help
+```
+
+Which prints:
+
+```
+...
+Available Commands:
+  can-play-move    Query canPlayMove
+...
+```
+
+</CodeGroupItem>
+<CodeGroupItem title="Can-play-move">
```

Comment:
```suggestion
<CodeGroupItem title="Can play move">
```
It was in reference to the command used in the content below. But if it has to be capitalised, we might as well make it a sentence.